rbac/rbac.go

```diff
@@ -0,0 +1,121 @@
+package rbac
+
+import (
+	"fmt"
+	"io"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Permission is an Observatorium RBAC permission.
+type Permission string
+
+const (
+	// Write gives access to write data to a tenant.
+	Write Permission = "write"
+	// Read gives access to read data from a tenant.
+	Read Permission = "read"
+)
+
+// Role describes a set of permissions to interact with a tenant.
+type Role struct {
```

Comment:
I think we talked about adding `resource` here, to control what exactly about this tenant the role is allowed to do.